src/recon_buildblock/BinNormalisationFromGEHDF5.cxx

```diff
@@ -323,14 +322,14 @@ read_norm_data(const string& filename)
     m_input_hdf5_sptr->initialise_efficiency_factors();
 
     // Do the reading using a buffer.
-    unsigned int total_size = scanner_ptr->get_num_rings()*scanner_ptr->get_num_detectors_per_ring();
+    unsigned long total_size = (scanner_ptr->get_num_rings()-1)*(scanner_ptr->get_num_detectors_per_ring()-1);
     stir::Array<1, float> buffer(0, total_size-1);
+    buffer.fill(0.0);
     m_input_hdf5_sptr->read_efficiency_factors(buffer);
     // Aparently GE stores the normalization factor and not the "efficiency factor", so we just need to invert it. 
     // Lambda function, this just applies 1/buffer and stores it in efficiency_factors 
-    std::transform(buffer.begin(), buffer.end(),efficiency_factors.begin_all(), [](const float f) { return 1/f;} );
+    std::transform(buffer.begin(), buffer.end(),efficiency_factors.begin_all(), [](const float f) { return 1/(f+0.000001);} );
```

Comment:
Did you really need this epsilon? I can see a zero efficiency occurring (if a detector is out), but as GE actually stores 1/efficiencies in the buffer, I can *not* see why the buffer would contain any zeroes (as this would correspond to an infinitely efficient detector!)

Reply:
See avobe. Debugging changes unreversed. My mistake